server/README.md

```diff
@@ -1,19 +1,8 @@
-## Common Setup
-```sh
-$ python3 -m venv env
-$ source env/bin/activate
-$ pip install -r requirements.txt
-```
-
 ## How to Start Server
-
-```sh
-$ export FLASK_APP=halomod
-$ export FLASK_ENV=development
-$ flask run
-```
+sh run.sh --dev
 
 ## How to Run Tests
-```sh
-$ python -m pytest
-```
+sh run.sh --test
+
+## How to run lint
+sh run.sh --lint
```

Comment:
It might be nice to have these in code blocks like they were before.

Reply:
Is reading run.sh not good enough? Why do we need a copy of the run.sh code in the readme? That also means if run.sh changes, we'll have to update the readme. It makes it not "dry"

Reply:
You already wrote the code. All that is needed is 3 back ticks. For example:

## How to run lint
```
sh run.sh --lint
```

That is all that is being said here.

Reply:
Oh those three backticks. I thought you meant something else.